Doxyfile

```diff
@@ -48,13 +49,13 @@ PROJECT_NAME           = "Cockatrice"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = 2.11
+PROJECT_NUMBER         = $(COCKATRICE_REF)
```

Comment:
how is this treated when running outside ci? what about manual triggers from the main branch?